lib/ui/dart_runtime_hooks.cc

```diff
@@ -223,4 +226,51 @@ void ScheduleMicrotask(Dart_NativeArguments args) {
   UIDartState::Current()->ScheduleMicrotask(closure);
 }
 
+void LookupClosure(Dart_NativeArguments args) {
+  Dart_Handle closure_name = Dart_GetNativeArgument(args, 0);
+  Dart_Handle library_name = Dart_GetNativeArgument(args, 1);
+  Dart_Handle library;
+  if (library_name == Dart_Null()) {
+    library = Dart_RootLibrary();
+  } else {
+    library = Dart_LookupLibrary(library_name);
+    DART_CHECK_VALID(library);
+  }
+  Dart_Handle closure = Dart_GetClosure(Dart_RootLibrary(), closure_name);
```

Comment:
Dart_RootLibrary() -> library? Otherwise library is unused.

Reply:
Yes, will fix.